multiprofit_task.py

```diff
@@ -130,11 +166,19 @@ class MultiProFitConfig(pexConfig.Config):
                                                                    "each model's best fit")
     outputLogLikelihood = pexConfig.Field(dtype=bool, default=True, doc="Whether to save the log likelihood "
                                                                         "of each model's best fit")
-    outputRuntime = pexConfig.Field(dtype=bool, default=True, doc="Whether to save the runtime of each "
-                                                                  "model")
+    outputRuntime = pexConfig.Field(dtype=bool, default=True, doc="Whether to save the runtime of each model")
+    pathCosmosGalsim = pexConfig.Field(
+        dtype=str, optional=True,
+        doc="Path containing COSMOST HST F814W-based real_galaxy_catalog_25.2.fits and "
+            "real_galaxy_PSF_images_25.2_n[1-88].fits (https://zenodo.org/record/3242143)"
+    )
```

Comment:
Another option that looks like it involves behavior that isn't legal in a `PipelineTask`.

Reply:
To be honest, I haven't used this option in years and I'm not sure it even works anymore, so I'll probably just remove it. The other `fitHstCosmos`-related options also do I/O through some cutout code, which you wouldn't know without digging further into the relevant functions.
This isn't urgent but are there legal ways of using external catalogs/images, or of ingesting say, coadds & PSFs of arbitrary provenance (like COSMOS-HST?).

Reply:
If you can ingest them into the butler, then it becomes legal.  Whether that's hard depends largely on whether you can easily cook up a data ID for them that only uses our existing data ID key-value pairs ("no data ID because there can only be one" may be a valid option).